app/models/game.rb

```diff
@@ -0,0 +1,47 @@
+class Game < ActiveRecord::Base
+  has_many :pieces
+  has_many :players
+
+  after_create :populate_board!
+
+  def populate_board!
+
+
+    # Black Pieces of the board
+    (0..7).each do |i|
+      Pawn.create(game_id: id, x_position: i, y_position: 1, color: "black")
+    end
+
+    Rook.create(game_id: id, x_position: 0, y_position: 0, color: "black")
```

Comment:
It does not need to be a symbol but I don't believe id will work anyway as 'id' will probably be an undeclared variable.  The problem here is that the pieces created here need to belong to the game.  Look at the rails associations that were set up for Game and Piece and use different syntax to create each piece so that it automatically belongs to the game.  The correct syntax should be `game.pieces.create(type: 'Rook', color: 'black, x_position: 0, y_position: 0)'.  This will also be revealed when a suite of unit tests are added.